data/scripts.js

```diff
@@ -647,9 +649,7 @@ let BattleScripts = {
 			// Modifies targetsCopy (which is why it's a copy)
 			[moveDamage, targetsCopy] = this.spreadMoveHit(targetsCopy, pokemon, move, moveData);
 
-			if (!moveDamage.some(val => val !== false)) {
-				break;
-			}
+			if (!moveDamage.some(val => val !== false)) break;
 			nullDamage = false;
 
 			for (let i = 0; i < damage.length; i++) {
```

Comment:
Look, shadowing!

Reply:
Didn’t I make this `j` in another commit?

Reply:
No, that was another loop.

Reply:
Guess there’s no need if I’m renaming the first `i`

Reply:
Actually it was this loop.